src/lib/DataTableHeader/DataTableRow.tsx

```diff
@@ -42,23 +83,22 @@ export function DataTableRow<T, TRouteNames>({
             column.cellStyle instanceof Function
               ? column.cellStyle({ key: keyValue, row: record, value: deepValue })
               : column.cellStyle ?? undefined;
-
           if (column.enumValues && !Number.isNaN(deepValueInt) && column.enumValues.filter((c) => c.value === deepValueInt).length > 0)
             return (
-              <td key={key} style={style}>
+              <td key={key} style={{ ...setStyle(record, rowHighlight), ...style }}>
```

Comment:
I would suggest to move this duplicate code to a `const` variable before the return statement. 

For example 
`const cellStyle=...`

Reply:
Fixed!